src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -69,6 +71,11 @@ private ConversationStore(PersistentStorageAgent persistentStorageAgent) {
     return conversations;
   }
 
+  // latest
+  public List<Conversation> getLatestConversations() {
+    return conversations; // all for now
```

Comment:
If you plan to change this later, consider adding a TODO, like so:

// TODO(Hannah): Update this to return N most recent conversations.